src/main/java/net/shadowmage/ancientwarfare/structure/init/AWStructureBlocks.java

```diff
@@ -13,29 +13,13 @@
 import net.shadowmage.ancientwarfare.core.item.ItemBlockBase;
 import net.shadowmage.ancientwarfare.core.util.InjectionTools;
 import net.shadowmage.ancientwarfare.structure.AncientWarfareStructure;
-import net.shadowmage.ancientwarfare.structure.block.BlockAdvancedLootChest;
-import net.shadowmage.ancientwarfare.structure.block.BlockAdvancedSpawner;
-import net.shadowmage.ancientwarfare.structure.block.BlockDraftingStation;
-import net.shadowmage.ancientwarfare.structure.block.BlockFirePit;
-import net.shadowmage.ancientwarfare.structure.block.BlockGateProxy;
-import net.shadowmage.ancientwarfare.structure.block.BlockGoldenIdol;
-import net.shadowmage.ancientwarfare.structure.block.BlockProtectionFlag;
-import net.shadowmage.ancientwarfare.structure.block.BlockSoundBlock;
-import net.shadowmage.ancientwarfare.structure.block.BlockStructureBuilder;
-import net.shadowmage.ancientwarfare.structure.block.BlockStructureScanner;
-import net.shadowmage.ancientwarfare.structure.block.BlockTotemCube;
-import net.shadowmage.ancientwarfare.structure.block.BlockTotemPart;
+import net.shadowmage.ancientwarfare.structure.block.*;
```

Comment:
Make sure to import code style from repo root / code_format/code_format.xml in IDEA File>Settings and then Editor>Code Style on the right next to the scheme drop down there's a cog where you can do import scheme and then the IDEA format and select that xml above. that way these imports won't get shortened to a wildcard character but there will be individual class imports instead in the future.

Once you have the code style just go here and you should get the proper imports by pressing Ctrl + Alt + O (organize imports)